auctions/models.py

```diff
@@ -434,35 +421,34 @@ def notify_approved_claim(sender, instance, created, **kwargs):
     votes_needed = claim.offers.count()
 
     if claim.num_rejections == votes_needed:
+        email_template = get_template('../templates/email/claim_reject.html')
+
         current_site = Site.objects.get_current()
-        # TODO: make a nicer HTML email template
-        CLAIM_REJECTED_EMAIL_STRING = """
-        Your claim for {url} has been rejected.
-        https://{site}
-        """
+        email_context = ({
+            'url': claim.issue.url,
+            'site': current_site,
+        })
+        message = email_template.render(email_context)
         send_mail(
             "[codesy] Your claimed has been rejected",
-            CLAIM_REJECTED_EMAIL_STRING.format(
-                url=claim.issue.url,
-                site=current_site,
-            ),
+            message,
             settings.DEFAULT_FROM_EMAIL,
             [claim.user.email]
         )
 
     if votes_needed == claim.num_approvals:
+        email_template = get_template('../templates/email/claim_approved.html')
+
         current_site = Site.objects.get_current()
-        # TODO: make a nicer HTML email template
-        CLAIM_APPROVED_EMAIL_STRING = """
-        Your claim for {url} has been approved.
-        https://{site}
-        """
+        email_context = ({
+            'url': claim.issue.url,
+            'site': current_site,
+        })
+        message = email_template.render(email_context)
+
         send_mail(
             "[codesy] Your claimed has been approved",
```

Comment:
typo: `Your claimed` should be `Your claim` ... and can we include the claim amount here again to encourage the recipient to open the email?